nbdev/showdoc.py

```diff
@@ -258,14 +314,20 @@ def show_doc(elt, doc_string=True, name=None, title_level=None, disp=True, defau
     title_level = title_level or (default_cls_level if inspect.isclass(elt) else 4)
     doc =  f'<h{title_level} id="{qname}" class="doc_header">{name}{source_link}</h{title_level}>'
     doc += f'\n\n> {args}\n\n' if len(args) > 0 else '\n\n'
+    s = ''
     if doc_string and inspect.getdoc(elt):
         s = inspect.getdoc(elt)
         # show_doc is used by doc so should not rely on Config
-        try: monospace = (get_config().get('monospace_docstrings') == 'True')
+        try: monospace = (Config().get('monospace_docstrings') == 'True')
         except: monospace = False
         # doc links don't work inside markdown pre/code blocks
         s = f'```\n{s}\n```' if monospace else add_doc_links(s, elt)
         doc += s
+    if len(args) > 0 and "arg" not in s.lower():
+        try:
+            doc += f"\n\n{_format_args(elt)}"
+        except:
```

Comment:
This `except` clause should be more limited, and documented. Even better if you can avoid it

Reply:
Now that it's been iterated over more, this isn't really needed. It was there as a catch-all in case `docments` failed, but docments really *shouldn't* fail given its nature. 

What we may want to do instead (and this goes with the displaying/figuring out if it's docments or not) is to include `show_docments` as a boolean at the end of `show_doc` and `doc`, that is set to `True` by default. WDYT?

Reply:
Went with that route as well as an explicit warning:

```python
    if len(args) > 0 and show_docments:
        try:
            doc += f"\n\n{_format_args(elt)}"
        except AttributeError:
            print(f"Warning: `docments` annotations are unavailable for {qual_name(elt)} and will not be shown.")
            pass
```
The issue with this is something about the `DocsTestClass` it will not let `docments` grab or do anything, it raises an issue. Not 100% sure why that's the case, but opt'd for just an explicit warning

Reply:
Found an example where this happens in case it's helpful: 
```
import numpy as np
show_doc(np.arange)
```

Reply:
I don't actually see where that's ever defined, so it may be a meta programming issue? I think just having the `doesn't support XYZ for docments` is fine as something to revert to, and just show what is normally there.

@jph00 I think what we should do in this case is have monospace be toggleable so it renders well. eg:
```python
show_doc(np.arange, show_docments=False, monospace=True)
```
is how we "fix" the numpy issue. Basically if folks want to keep using that type of documentation string, we don't markdown format it

Reply:
Solution fixed. The issue was builtin modules cannot have source, so `getsource()` failed. Put in an explicit print about it.